commercetools-models/src/test/java/io/sphere/sdk/shippingmethods/queries/ShippingMethodsByLocationGetIntegrationTest.java

```diff
@@ -20,7 +20,7 @@
     public void execution() throws Exception {
         withShippingMethodForGermany(client(), shippingMethod -> {
             final SphereRequest<List<ShippingMethod>> sphereRequest =
-                    new VrapRequestDecorator<>(ShippingMethodsByLocationGet.of(CountryCode.DE), "response");
```

Comment:
Does it actually fail if the expansion path is not recognized? In a normal situation I know it fails silently, returning the query results as nothing happened, so it wouldn't fail. But as it's wrapped in this `VrapRequestDecorator` maybe it's checking it against the RAML specification?

Reply:
This decorator is used to disable the response validation of our vrap proxy running on TeamCity and it then just pass through the response from our backend. And we have to disable the validation here because the same uri is used by multiple `GET` endpoints which return different types: `List<ShippingMethod>` vs `PagedQueryResult<ShippingMethod>`. And this can't be expressed in RAML and it's considered bad RESTful practise too 😉

Reply:
So that means that the test won't fail if `m.zones()` doesn't return the proper expansion path, right? If that's the case, then I would add an assertion checking that the returned shipping methods have zones expanded. The same for the test with Tax Category.

Reply:
The `VrapRequestDecorator` doesn't have any effect on the test if it's disabled. But you're right that the test won't fail when the expansion doesn't work 👍 I already added asserts to ensure that the expansion works.